examples/json/server/main.go

```diff
@@ -83,6 +83,7 @@ func main() {
 			tch.NewInbound(channel, tch.ListenAddr(":28941")),
 			http.NewInbound(":24034"),
 		},
+		Interceptors: yarpc.Interceptors{requestLogInterceptor{}},
```

Comment:
It's not a wrapper. It's an alias for writing `[]transport.Interceptors{...}`. That's how you create slices. When there are more than one interceptors or filters, this becomes,

```go
yarpc.Interceptors{
    interceptor1,
    interceptor2,
    interceptor3,
}
```